plugins/workunitservices/workunitservices.cpp

```diff
@@ -526,37 +526,48 @@ WORKUNITSERVICES_API char * wsWUIDdaysAgo(unsigned daysago)
     return getWUIDdaysAgo(ret,(int)daysago).detach();
 }
 
+class WusTimeStampVisitor : public WuScopeVisitorBase
```

Comment:
@ghalliday It's  not clear what "Wus" means?

Reply:
WorkUnitService.   I will change to use Ws for consistency with the rest of the file.